### PR TITLE
Add test favorite icon

### PR DESCRIPTION
**Pull request by Anchello**

### CODE REVIEW

File: packages/shared/components/favorite-asset/favorite-icon/favorite-icon.tsx [hunk 1]
```diff
@@ -5,13 +5,6 @@ import { Icon } from "shared/components/icon/icon";
...
 import { ReactComponent as Favorite } from "./favorite.svg";
 
-interface IFavoriteIconProps {
-  id: string;
-  onClick?(id: string, selected: boolean): void;
-  selected: boolean;
-  className?: string;
-}
-
 class FavoriteIcon extends React.Component<IFavoriteIconProps> {
```
Comment (pavelzubov) [inline packages/shared/components/favorite-asset/favorite-icon/favorite-icon.tsx]: Change to PureComponent

File: packages/shared/components/favorite-asset/favorite-icon/favorite-icon.tsx [hunk 2]
```diff
@@ -34,4 +27,11 @@ class FavoriteIcon extends React.Component<IFavoriteIconProps> {
   }
 }
 
+interface IFavoriteIconProps {
```
Comment (pavelzubov) [inline packages/shared/components/favorite-asset/favorite-icon/favorite-icon.tsx]: Rename to Props

Comment (Anchello) [reply packages/shared/components/favorite-asset/favorite-icon/favorite-icon.tsx]: Done

File: packages/shared/components/favorite-asset/favorite-icon/favorite-icon.tsx (hunk 1 above)
Comment (Anchello) [reply packages/shared/components/favorite-asset/favorite-icon/favorite-icon.tsx]: Done

**Pull request merged**